src/schema/validator.js

```diff
@@ -80,9 +82,9 @@ export const validateAddon = (...args) => {
 // structure and can use object-destructuring at the root level
```

Comment:
Nit, I guess that this comment could use some tweak now that we have a better picture of why this parts need to use a different "merging" algorithm.

Reply:
done in dc639e2c